pkg/gcv/configs/config.go

```diff
@@ -491,6 +491,10 @@ func NewConfiguration(dirs []string, libDir string) (*Configuration, error) {
 		return nil, err
 	}
 
+	if len(unstructuredObjects) == 0 && len(regoLib) == 0 {
```

Comment:
The regoLib files don't do anything on their own, so we should only be checking for the unstructuredObjects.

Reply:
Fixed. I moved the logic into `loadUnstructured()` and removed the check on `regoLib`.